packages/core/src/delegation/Delegation.chain.ts

```diff
@@ -24,7 +24,7 @@ export async function getAttestationHashes(
 ): Promise<string[]> {
   const blockchain = await BlockchainApiConnection.getConnectionOrConnect()
   const encodedHashes = await blockchain.api.query.attestation.delegatedAttestations<
-    Vec<H256>
+    Option<Vec<H256>>
```

Comment:
I think we should replace all `H256` with `Hash`, at this point, right? At least for all things related to attestations.

Reply:
Can do, it's nothing but an alias anyway